### PR TITLE
Changes to transfer template and template definition.

### PR DESCRIPTION
**Pull request by UlysseFG**

### CODE REVIEW

File: backend/fms_core/tests/test_template_importers/sample_pooling_planning_test.py
Comment (nafiz1001) [inline backend/fms_core/tests/test_template_importers/sample_pooling_planning_test.py:1]: Ideally, the second index should be a constant but it's too late I guess

File: backend/fms_core/template_importer/row_handlers/transfer/transfer.py
Comment (nafiz1001) [inline backend/fms_core/template_importer/row_handlers/transfer/transfer.py:1]: We need better variable names, but I'll assume it's good ;)

File: backend/fms_core/tests/test_template_importers/transfer_test.py
Comment (nafiz1001) [inline backend/fms_core/tests/test_template_importers/transfer_test.py:1]: We need better variable names, but I'll assume it's good if it passes ;)

**Review by nafiz1001 — APPROVED** [review]

File: backend/fms_core/tests/test_template_importers/sample_pooling_planning_test.py
Comment (UlysseFG) [reply backend/fms_core/tests/test_template_importers/sample_pooling_planning_test.py:1]: too late ? which index ? TWO = 2 ?

Comment (nafiz1001) [reply backend/fms_core/tests/test_template_importers/sample_pooling_planning_test.py:1]: More like
```python
self.assertEqual(csv_content[1][ROBOT_SRC_BARCODE], "Src1")
```

**Pull request merged**